### PR TITLE
Minor grammar fixes and improvements in README.md

### PR DESCRIPTION
**Pull request by Michael21107**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Hello there,<br>
 I am Scotty (Aka. Scottywonderful), I hope this finds you well and that you are having a great day.<br>
 I am pleased to announce that this is my first ever callout pack created for LSDPFR.<br>
 With this, I am excited to say that I have multiple ideas and updates for this pack in the future.<br>
-That's all from me, have fun, remember to think safe, act safe, be safe!<br>
+That's all from me, have fun, remember to think safe, act safe, and be safe!<br>
```
Comment (Scottywonderful) [inline README.md:57]: This is a good motto from a transportation company I use to do work with :) So I won't change this :)

File: README.md [hunk 2]
```diff
@@ -235,6 +235,6 @@ I would like to thank the following people for their help, ideas and contributio
...
 * ![natey1.](https://dcbadge.vercel.app/api/shield/651319300166254592?style=flat-square&?compact=true&?logoColor=blue) (Alpha and Beta testing for me :neutral_face:)<br> -->
 
-* Last but not least, <b><a href="https://discord.com/users/291193188956241921">Scottywonderful</a></b> (Yes, myself, because I made this mostly from scratch and did my own ideas in it as well :cowboy_hat_face:)<br>
+* Last but not least, <b><a href="https://discord.com/users/291193188956241921">Scottywonderful</a></b> (Yes, myself, because I made this mostly from scratch and did my ideas in it as well :cowboy_hat_face:)<br>
 <!--![Scottywonderful](https://dcbadge.vercel.app/api/shield/291193188956241921?style=flat-square&?compact=true&?logoColor=purple)<br>-->
-Need any support then please join our discord <a href="https://discord.gg/GKbEFJZvz4">https://discord.gg/GKbEFJZvz4</a><br>
\ No newline at end of file
+Need any support then please join our discord <a href="https://discord.gg/GKbEFJZvz4">https://discord.gg/GKbEFJZvz4</a><br>
```
Comment (Scottywonderful) [inline README.md:240]: No sure what change you did here as it shows there is a change but I don't see one.

**Review by Scottywonderful — CHANGES_REQUESTED** [review]
If you can just change the 1 thing I commented on in line 57 that would be great, otherwise this pull request will be accepted. Thank you for the grammar check -- I did ask my testers and other staff to check it before I released it but I guess they didn't and I clearly took their word for it :)

**Review by Scottywonderful — APPROVED** [review]
I will accept the changes but I will change back that one thing. That's the only thing I think that needs to change.

File: README.md (hunk 1 above)
Comment (Scottywonderful) [reply README.md:57]: I have decided to leave the and in the edit. Thanks for the feedback.

File: README.md (hunk 2 above)
Comment (Michael21107) [reply README.md:240]: I guess just GitHub being GitHub.

**Pull request merged**